src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValues.ts

```diff
@@ -47,6 +47,10 @@ export class DynamicFacetValues implements IDynamicFacetValues {
     return this.facetValues.filter(value => value.isSelected).map(({ value }) => value);
   }
 
+  private get selectedDisplayValues() {
```

Comment:
When we add the possibility to exclude values, this might "slip" and we might forget about adding the correct logic here ( for `.isExcluded` ).

Should we instead use `! .isIdle` ?

Reply:
When we have excluded we might want to display those values separately and differently. E.g. : ~excluded value 1~